manifests/04-ca-proxy.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    config.openshift.io/inject-proxy-cabundle: "true"
```

Comment:
this also needs the create-only annotation so the CVO doesn't keep stomping it:
https://github.com/openshift/cluster-config-operator/blob/master/empty-resources/0000_05_config-operator_02_apiserver.cr.yaml#L6